service/routers/users.js

```diff
@@ -0,0 +1,50 @@
+import { Router } from 'express'
+import { User } from '../models/User.js'
+
+const usersRouter = Router()
+
+const getUsers = async (req, res) => {
+  if(req.user.role == "Admin" || req.user.role == "SuperUser") {
```

Comment:
You should not be using the role sent by the client. That can be faked. You should get the user from the database, a trusted source, and use that role.